### ISSUE TITLE
Add colorbar to instrument plots in tutorials

### ISSUE THREAD

**Issue by drannawatts** [xpsi-group/xpsi#403]
In the 'Instrument synergy' tutorial (and possibly elsewhere?) add a colorbar and an explanation of what the colors are.

**Comment by Basdorsman** [comment]
The RMF is a matrix gives the statistical distribution of how incident radiation is distributed to detection bins. You can check this by summing over detection bins in that RMF matrix and then the values at each incident energy should sum up to one. 

The ARF is a multiplicative product of the geometric area, which is energy independent and measured in units of cm2, and the quantum efficiency, which is the probability to convert a single incident photon into an observed detector count. (https://heasarc.gsfc.nasa.gov/docs/nicer/analysis_threads/arf-rmf/)

In conclusion the colorbar should be effective area times quantum efficiency per incident bin and per detection bin.

**Comment by denis-gc** [comment]
Correct also the units for plot of effective area, which should be cm^2. In the same plot, add labels  for NICER and XMM.
